portal/templates/home.html

```diff
@@ -3,16 +3,22 @@
 {% block content %}
 
   <h1>welcome to teacher home page</h1>
+  {% for course in courses %}
+  <li>{{ course['major'], course['name'], course['description'] }}</li>
```

Comment:
I think a more semantic way of doing this would be to put the list items in either a <ul> or <ol>